sonar-ws/src/main/protobuf/ws-commons.proto

```diff
@@ -72,17 +72,6 @@ enum RuleStatus {
   REMOVED = 3;
 }
 
-message User {
```

Comment:
have we quit the idea to have common format for elements? (components, users come to mind)